lib/reverse_sentence.rb

```diff
@@ -1,6 +1,24 @@
 # A method to reverse the words in a sentence, in place.
```

Comment:
Note on comprehension questions:  All the basic sorting algorithms have O(n^2) time complexity.  However MergeSort is O(n log n)